Classes/DDFileLogger.m

```diff
@@ -433,6 +434,28 @@ - (NSString *)newLogFileName {
     return [NSString stringWithFormat:@"%@ %@.log", appName, formattedDate];
 }
 
+- (void)modifyLogFileCreateDateIfNeed:(NSString *)filePath
+{
+    NSDate* fileCreateDate = [NSDate date];
+    if (_lastLogFileCreateDate == nil) {
+        DDLogFileInfo *logFileInfo = [self sortedLogFileInfos].firstObject;
+        if (logFileInfo) {
+            _lastLogFileCreateDate = logFileInfo.creationDate;
+        }
+    }
+    if (_lastLogFileCreateDate && (int)_lastLogFileCreateDate.timeIntervalSince1970 >= (int)fileCreateDate.timeIntervalSince1970) {
+        fileCreateDate = [_lastLogFileCreateDate dateByAddingTimeInterval:1];
```

Comment:
Yes, I understand that there may be some edge cases as this, but your pull request does not fix the problem. As I said:

> If you're creating so many log files fileCreateDate + 1 may also collision so this won't be very useful.